client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -263,6 +268,18 @@ class Upload extends React.Component {
 		);
 	}
 
+	renderNotAvailable() {
+		return (
+			<EmptyContent
+				title={ this.props.translate( 'Sorry, themes upload is not yet available for multi-site installations.' ) }
+				line={ this.props.translate( 'Open your site:' ) }
```

Comment:
True, that's better. It's just that it was such an edge case that we didn't focus on the message too much as you noticed.

Let's do:

> **Not available for multi site**
> Use the WP Admin interface instead
> Button: "Open WP Admin"

Works?